docs/developer/index.rst

```diff
@@ -333,13 +333,9 @@ The ``sherpa/conftest.py`` file contains general-purpose testing
 routines, fixtures, and configuration support for the test suite.
 To add a new command-line option:
 
-* add to the ``pytest_addoption`` routine, to add the option;
+* to add generic options modify the ``pytest_addoption``;
 
-* add to ``pytest_collection_modifyitems`` if the option adds
-  a new mark;
-
-* and add support in ``pytest_configure``, such as registering
-  a new mark.
+* for new marks, add to ``markers`` in ``pytest.ini``.
```

Comment:
I've never felt happy with how we/I documented this.

Your change says to add new markers to ``pytest.ini`` but below you change ``pytest_addoption`` in ``sherpa/conftest.py``.

So am I getting confused by what `mark` means?

Reply:
Yeah no. It's me. Pytest ha sa way [to add marks in pytest.ini](https://docs.pytest.org/en/stable/how-to/mark.html#registering-marks) which is simpler than what we do today and which I tried to use. However, by default it always runs all tests, unless a user types `pytest -m "not speed". In other words, ther is no way to set the default to False.
So, I reverted conftest, but not the docs.